hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -58,7 +60,7 @@ import Test.QuickCheck (
   property,
   suchThat,
   vectorOf,
-  withMaxSuccess,
+  withMaxSuccess, getPositive
```

Comment:
No formatter in use?

Reply:
seems so. I re-saved the file and it worked.